trace_viewer/core/timeline_track_view.html

```diff
@@ -202,6 +202,13 @@
       this.mouseModeSelector_.setKeyCodeForMode(m.ZOOM, '3'.charCodeAt(0));
       this.mouseModeSelector_.setKeyCodeForMode(m.TIMING, '4'.charCodeAt(0));
 
+      var _this = this;
+      this.mouseModeSelector_.setKeyCodeCondition(function() {
+        // Return false when FindControl is active so that MouseMode keyboard
+        // shortcuts are ignored.
+        return _this.listenToKeys_;
+      });
```

Comment:
You should be able to }.bind(this)); and then just use this inside the method. You then don't need the var _this = this;